src/Providers/Api/Request/GenericPaymentDataProvider.php

```diff
@@ -103,19 +103,20 @@ public function getSetOrderReferenceDetailsRequestData(
         string $paymentCode,
         string $workOrderId,
         string $amazonReferenceId,
-        string $currency,
         string $amount,
+        string $currency = null,
```

Comment:
Then you would have to adjust the php doc as well

Reply:
Done